components/greaselion/browser/BUILD.gn

```diff
@@ -16,9 +14,11 @@ source_set("browser") {
   deps = [
     "//base",
     "//brave/components/brave_component_updater/browser",
+    "//chrome/browser/extensions:extensions",
+    "//chrome/common",
     "//content/public/browser",
     "//content/public/common",
-    "//chrome/common",
+    "//extensions/browser:browser_sources",
```

Comment:
@bridiver I've tried to avoid touching this upstream file. But, some greaselion files have a dependency with extension/browser. For example, extensions/browser/extension_system.h, extensions/browser/extension_registry.h. So, I need to add this dependency here. Do you have any idea to avoid adding this dependency?

Reply:
you only need `//extensions/browser` for these
```
group("browser") {
  public_deps = [
    "//extensions/browser:browser_context_keyed_service_factories",
    "//extensions/browser:browser_sources",
    "//extensions/browser/api:api_registration",
  ]

  # Includes all API implementations and the ExtensionsApiClient
  # interface. Moving an API from src/chrome to src/extensions implies
  # it can be cleanly disabled with enable_extensions=false.
  public_deps += [ "//extensions/browser/api" ]
}
```

Reply:
Thanks, I've only considered to add "//extensions/browser:browser_sources". It works.